### PR TITLE
Fix about main

### PR DESCRIPTION
**Pull request by c0smic0wl**
- Add Countdown Section for ICO.
- Add Medium Link.
- Fix some responsible css issues.

### CODE REVIEW

File: app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './crowdSaleContainer.scss';
+import Icon from '../../../icons';
+import { updateCountdown } from './actions';
+
+function mapStateToProps(appState) {
+  return {
+    // pass
```
Comment (c0smic0wl) [inline app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx]: Ah.. I'll remove it.

File: app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    );
+  }
+
+  updateTimer() {
+    const { dispatch } = this.props;
+    dispatch(updateCountdown());
+    // const dueDate = new Date('2017-08-15T14:00:00.000Z');
+    // this.props.due
+    // console.log('hi');
+    // console.log(this.props.aboutCrowdSale.get('dueDate'));
```
Comment (parkkoon) [inline app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx]: Please remove this, too.

**Review by parkkoon — APPROVED** [review]
👍

File: app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+class CrowdSaleContainer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.updateTimer = this.updateTimer.bind(this);
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(this.updateTimer, 500);
```
Comment (fish-sushi) [inline app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx:21]: 👍

File: app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
```
Comment (fish-sushi) [inline app/javascripts/components/about/crowdSaleContainer/crowdSaleContainer.jsx:25]: 💯

File: app/javascripts/icons/crowdsale-ticket.svg [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
..."M1566 542.599v48.545c0 13.728-11.128 24.856-24.856 24.856h-341.78c-.297-34.41-30.698-62.219-68.158-62.219s-67.861 27.81-68.158 62.219H24.856C11.128 616 0 604.872 0 591.144V542.6c.414.01.829.015 1.245.015 25.437 0 46.059-18.42 46.059-41.141 0-22.722-20.622-41.14-46.06-41.14-.415 0-.83.004-1.244.014v-40.058c.414.01.829.014 1.245.014 25.437 0 46.059-18.42 46.059-41.14 0-22.722-20.622-41.141-46.06-41.141-.415 0-.83.005-1.244.014v-40.058c.414.01.829.015 1.245.015 25.437 0 46.059-18.42 46.059-41.141 0-22.721-20.622-41.14-46.06-41.14-.415 0-.83.004-1.244.014v-40.058c.414.01.829.014 1.245.014 25.437 0 46.059-18.42 46.059-41.14 0-22.722-20.622-41.141-46.06-41.141-.415 0-.83.005-1.244.014v-68.56C0 11.129 11.128 0 24.856 0h1038.192l-.002.508c0 34.643 30.516 62.727 68.16 62.727s68.16-28.084 68.16-62.727c0-.17 0-.339-.002-.508h341.78C1554.872 0 1566 11.128 1566 24.856v68.56c-24.862.589-44.814 18.776-44.814 41.126 0 22.35 19.952 40.536 44.814 41.126v40.058c-24.862.59-44.814 18.776-44.814 41.126 0 22.35 19.952 40.537 44.814 41.126v40.058c-24.862.59-44.814 18.777-44.814 41.126 0 22.35 19.952 40.537 44.814 41.127v40.058c-24.862.59-44.814 18.776-44.814 41.126 0 22.35 19.952 40.537 44.814 41.126z"/>
```
Comment (fish-sushi) [inline app/javascripts/icons/crowdsale-ticket.svg]: Change id

**Review by fish-sushi — CHANGES_REQUESTED** [review]

**Pull request merged**